src/components/EventResults.js

```diff
@@ -0,0 +1,33 @@
+import { Link } from "react-router-dom";
+
+const EventResults = ({ events }) => {
+  return (
+    <section>
+      {events.length === 0 ? (
+        <h3> Discover the Hottest Music Events Near You!</h3>
+      ) : events.length > 0 ? (
```

Comment:
if i am not wrong, i think you can get rid of ```js events.length > 0 ``` and just keep the : so that would look like;

```js
{events.length === 0 ? (
  <h3> </h3>
) :  ( you display results here)
```
and get rid of the :null on line 28